python/pyproject.toml

```diff
@@ -1,3 +1,51 @@
+[build-system]
+requires = ["setuptools>=61.0", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "gherkin-official"
+version = "29.0.0"
+description = "Gherkin parser (official, by Cucumber team)"
+readme = "README.md"
+requires-python = ">=3.8"
+authors = [
+    { name = "Cucumber Ltd and Björn Rasmusson", email = "cukes@googlegroups.com" }
+]
+
+dependencies = [
+  "typing_extensions>=4",
+]
+keywords = ["gherkin", "cucumber", "bdd"]
+classifiers = [
+    "License :: OSI Approved :: MIT License",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12"
+]
+
+[project.urls]
+Homepage = "https://github.com/cucumber/gherkin"
+Repository = "https://github.com/cucumber/gherkin.git"
+Download = "https://pypi.python.org/pypi/gherkin-official"
+Issues = "https://github.com/cucumber/gherkin/issues"
+Changelog = "https://github.com/cucumber/gherkin/releases"
+
+[tool.setuptools.package-data]
+gherkin = ["gherkin-languages.json"]
+
+[tool.setuptools.packages.find]
+include = ["gherkin", "gherkin.pickles", "gherkin.stream"]
```

Comment:
BETTER: Use the `find` mechanism that is described in [setuptools docs: Find simple packages](https://setuptools.pypa.io/en/latest/userguide/package_discovery.html#finding-simple-packages)
* BLOCKER: In your case the `gherkin/scripts/` directory would have been missing.

Reply:
actually, I'm even more inclined to not include the `scripts/` folder in the distribution at all.

The reason is that these scripts are only used internally, as there is absolutely no API or documentation about these. Hell, I can't even figure out what the `gherkin` script does at all, since there is no documentation.

I really doubt that any downstream user is using these. If anything, we are just polluting their PATH with this irrelevant `gherkin` executable.

In the remote hypothesis that a downstream user was relying on this, we can always release a hotfix with the script included in the distribution.

Reply:
done in d992ac46